test/MezzioInstallerTest/HomePageResponseTest.php

```diff
@@ -174,7 +174,7 @@ public function testHomePageHtmlResponseContainsExpectedInfo(
         self::assertEquals(200, $response->getStatusCode());
 
         // Test response content
-        $html = (string) $response->getBody()->getContents();
+        $html = $response->getBody()->getContents();
```

Comment:
this is wrong. `getContents()` returns from current pointed location to the end. It needs to be `__toString()` or string cast for the body

Reply:
There is another usage like that on line 272

Reply:
[StreamInterface::getContents()](https://github.com/php-fig/http-message/blob/402d35bcb92c70c026d1a6a9883f06b2ead23d71/src/StreamInterface.php#L143) returns a string

Reply:
Doing `getContents()` on a new stream twice will return full content first and empty string second time. It returns remainder of the content from current pointer location.

`__toString()` does rewind first.